templates/prometheus.go

```diff
@@ -148,12 +151,23 @@ var PrometheusTemplate = promv1.Prometheus{
 				},
 				WriteRelabelConfigs: []promv1.RelabelConfig{
 					{
-						SourceLabels: []string{"__name__"},
-						Regex:        strings.Join(metrics[:], "|"),
+						SourceLabels: []string{"__name__", "alertname"},
```

Comment:
maybe I'm missing something, how would the job `alertname` has metrics attached to it?

If possible could you pls provide the source metrics before transformation/match?

Reply:
This might help with the explanation
![image](https://user-images.githubusercontent.com/5047914/181681046-ce8d39a2-f74b-4b52-9ecc-682568d50063.png)
![image](https://user-images.githubusercontent.com/5047914/181681137-909ceb9f-a542-45b1-bc42-89905c0aa62b.png)
![image](https://user-images.githubusercontent.com/5047914/181681190-8a86c7ff-8df7-42fe-923b-83c4af6395a6.png)

Reply:
- quite good! we are including `separator` in `regex` itself
- nevertheless, understood the outcome, thanks!

Reply:
Yes if no seperator is provided the default becomes ';'

Reply:
@ezio-auditore @leelavg 
The example above has an error in it:
`(ALERTS;a|b)` in regexp is `ALERTS;a` or `b` which is not what we want.
The regexp should be `(ALERTS;(a|b))` which translate to `ALERTS:a` or `ALERTS:b`.
Forutently enough the code does reflect the correct regexp.